pkg/config/config.go

```diff
@@ -39,6 +39,7 @@ type Config struct {
 	MaxBackoffMS        int // Maximum backoff in ms to wait after db connection error
 	PodName             string
 	PodNamespace        string
+	ResyncPeriod        int
```

Comment:
Can we change to milliseconds for consistency with other times?
```suggestion
	ResyncPeriod        int // Time in MS for the clusters informer. Default: 15 min.
```

Reply:
Updated.